tests/integration/models/reddit/test_subreddit.py

```diff
@@ -387,7 +400,7 @@ def test_delete(self, _):
                 'TestSubredditFlairTemplates.test_delete'):
             template = list(self.subreddit.flair.templates)[0]
             self.subreddit.flair.templates.delete(
-                template['flair_template_id'])
```

Comment:
Yes, but not on our end. Reddit changed the response of the endpoint we used previously. At the time the cassette for this test was recorded, Reddit's response to the `r/{subreddit}/api/flairselector/` endpoint was something of this shape:

```python
[{'flair_css_class': '',
  'flair_position': 'right',
  'flair_template_id': '0f571f6e-68f7-11e6-b114-0e745a265c0d',
  'flair_text': '',
  'flair_text_editable': True},
 {'flair_css_class': '',
  'flair_position': 'right',
  'flair_template_id': '12d00200-68f7-11e6-a565-0e3ec6006e2f',
  'flair_text': 'aaa',
  'flair_text_editable': True}]
```

Since then, however, they changed the endpoint to return a shape like this:

```
[{'background_color': '#000000',
  'id': '64f9e99a-1f4b-11e8-a44c-0e0337b0567e',
  'mod_only': False,
  'richtext': [{'e': 'text', 't': 'PRAW updated'}],
  'text': 'PRAW updated',
  'text_color': 'dark',
  'text_editable': False,
  'type': 'richtext'},
 {'background_color': '',
  'id': '81e77ec8-143c-11e9-95f6-0e576430b542',
  'mod_only': False,
  'richtext': [{'e': 'text', 't': 'Old use flair'}],
  'text': 'Old use flair',
  'text_color': 'dark',
  'text_editable': False,
  'type': 'richtext'}]
```

This is the same shape that the new endpoint (`r/{subreddit}/api/user_flair_v2`) returns.

So, yes, it is a breaking change, but it doesn't originate from PRAW, and any PRAW users who encountered it presumably have already updated their projects.

Reply:
As a test, try `pprint(list(reddit.subreddit('redditdev').flair.templates))` in the current release of PRAW. You'll see the new shape with `'id'`.